src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  // Split the input string by semicolons to get individual declarations
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    // Trim whitespace from the declaration
+    const trimmedDeclaration = declaration.trim();
+
+    // Ignore empty declarations
+    if (!trimmedDeclaration) {
+      return;
+    }
+
+    // Split each declaration into property and value
+    const [property, value] = trimmedDeclaration
+      .split(':')
+      .map((part) => part.trim());
```

Comment:
The current implementation assumes that each declaration will have exactly one colon. Consider adding error handling for cases where a declaration might be malformed, such as missing a colon or having multiple colons.